client/src/components/GraphViewerComponent/GraphViewerComponent.js

```diff
@@ -230,17 +230,14 @@ class GraphViewerComponent extends React.Component {
       if (this.cyRef.current) {
         this.cyRef.current.clearTwins();
       }
+
       await apiService.query(query, async data => {
         if (data.twins.length > 0) {
           this.setState({ couldNotDisplay: false, noResults: false, relationshipsOnly: false });
           this.cyRef.current.addTwins(data.twins);
           await this.cyRef.current.doLayout();
           data.twins.forEach(x => allTwins.push(x));
           this.updateProgress();
-
-          if (data.other.length > 0) {
```

Comment:
Is this the right place for the fix? I wonder why this was here...

I would've expected to see a change to the logic around adding the relationships. Presumably there is a check somewhere that blocks querying/adding edges to the graph if couldNotDisplay is true?

Reply:
There was a reason for this, I am trying to dig out the email from Christian. Funnily enough, he was the one that reported it not showing the relationships. But I remember there was a reason why he didn't want to

Reply:
Thanks. I seem to recall data was expected to end up in either twins or other. Rather than both. Keep in mind the twins/other split is something that we do in the ApiService so maybe our logic there isn't quite right?